recipes/openamundsen/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "openamundsen" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openamundsen-{{ version }}.tar.gz
+  sha256: cfcc305352be6cacb0c07f923b2b646f25481e4a86da595d021e25631e13fca9
+
+build:
+  number: 0
+  skip: true   # [py<37]
+  entry_points:
+    - openamundsen = openamundsen.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - cerberus
+    - importlib-metadata  # [py<38]
+    - loguru
+    - munch
+    - netcdf4
+    - numba >=0.50.1
+    - numpy
+    - pandas >=1.1.0
+    - pyproj
+    - python
```

And just unconditionally depend on `importlib-metadata`:
```suggestion
  host:
    - pip
    - python >=3.7
  run:
    - cerberus
    - importlib-metadata
    - loguru
    - munch
    - netcdf4
    - numba >=0.50.1
    - numpy
    - pandas >=1.1.0
    - pyproj
    - python >=3.7
```